persist/sqlite/v2consensus_refactored_test.go

```diff
@@ -2545,3 +2547,102 @@ func TestV2Metrics(t *testing.T) {
 
 	assertMetrics(metricsGenesis)
 }
+
+func BenchmarkV2Transactions(b *testing.B) {
+	n := newTestChain(b, false, nil)
+
+	// add a bunch of random transactions that are either empty, contain arbitrary
```

Comment:
`nit` Instead of saying "a bunch" we usually add a constant like `dbSize` or `nElements` or something that defines how big the setup is, so in your case this could be `const nTransactions = 1e6`.